backend/backend/charge_rent.py

```diff
@@ -0,0 +1,47 @@
+"""Module enables users to be charged rent
+   when they land on owned properties
+"""
+
+import json
+import sys
+import cgitb
+from backend.player import Player
+from backend.game import get_games
+from backend.properties import Property
+
+cgitb.enable()
+
+
+def charge_rent(source=sys.stdin, output=sys.stdout):
+    """Entry point for a player to be charged
+       rent and property owner gains rent amount
+    """
+
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request["player_id"]
+    games = get_games()
+    game_id = None
+
+    # Access the game the player is playing in
+    # and the position the player is on the board
+    with Player(player_id) as player:
+        position = player.board_position
+        for game in games:
+            if player.username in games[game]:
+                game_id = game
+                break
+
+        # Accesses owner and rent of property; if the games's
+        # current turn's player doesn't own it, charge the player
+        # and increase the property owner's balance
+        with Property(position, game_id) as propertie:
```

Comment:
There's a typo with `propertie` it should be `property` also on the lines below.

Reply:
I named it 'propertie' as 'property' is a built-in in python but I can change it to 'property'!

Reply:
Travis didn't like having to use 'property' so I reverted back.

Reply:
I think the guideline for variable names that clash with built-ins is a single trailing underscore so `property_` would be better